### PR TITLE
Prefs Read/Write Fix

### PR DESCRIPTION
**Pull request by gsmc-alx**
Fix to preference read/write, following removal of plistlib.readPlist() and plistlib.writePlist() from Python 3.8.

### CODE REVIEW

File: pkgroot/usr/local/outset/outset [hunk 1]
```diff
@@ -611,7 +616,8 @@ def main():
                 prefs["override_login_once"].update(override_items)
             else:
                 prefs["override_login_once"] = override_items
-            plistlib.writePlist(prefs, outset_preferences)
+            with open(outset_preferences, 'wb') as fp:
+                prefs = plistlib.dump(fp)
```
Comment (chilcote) [inline pkgroot/usr/local/outset/outset:620]: forgot to pass prefs to plistlib.dump here

Comment (chilcote) [reply pkgroot/usr/local/outset/outset:620]: change to plistlib.dump(prefs, fp)

File: pkgroot/usr/local/outset/outset [hunk 2]
```diff
@@ -624,7 +630,8 @@ def main():
                 script = os.path.join(login_once_dir, script)
                 if script in prefs["override_login_once"]:
                     del prefs["override_login_once"][script]
-            plistlib.writePlist(prefs, outset_preferences)
+            with open(outset_preferences, 'wb') as fp:
+                prefs = plistlib.dump(fp)
```
Comment (chilcote) [inline pkgroot/usr/local/outset/outset:634]: same here
change to plistlib.dump(prefs, fp)

**Review by chilcote — COMMENTED** [review]
Fixe those two and it looks reasonable to me. Thanks!

**Comment by gsmc-alx** [comment]
I’m running a system update on my laptop at the moment, so can’t fix those issues right now, unfortunately.

Could you maybe fix them at your end, if you have a moment?

**Comment by gsmc-alx** [comment]
Laptop update finished. Fixed those two issues. Sorry about that- silly mistakes...

**Comment by chilcote** [comment]
Looks good to me. I'll merge, and then we can get feedback from others to make sure no one finds any other edge cases. Thanks a lot for putting this together.

**Comment by gsmc-alx** [comment]
Great!

**Pull request merged**